docs/examples/finetuning/llm_judge/finetune_llm_judge.ipynb

```diff
@@ -0,0 +1,1248 @@
+{
```

Comment:
<p>is the plan to switch from PG essay to this pdf? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/8596/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Yeah, so that's what I started with, and then for some reason, I don't know why <code>DatasetGenerator</code> stopped working. Even after multiple kernel restarts and shutting down the jupyter notebook session, it wasn't working. I filed a bug <a href="https://linear.app/gpt-index/issue/LLAMA-597/[bug]-datasetgeneratorgenerate-questions-from-nodes-stalls-sometimes" target="_blank">ticket</a> in linear for future investigation. </p>

Reply:
used `WikipediaReader` instead now